src/DynamoCore/DynamoCore.csproj

```diff
@@ -18,6 +18,7 @@
     <Compile Remove="Graph\ConnectorPinModel.cs" />
   </ItemGroup>
   <ItemGroup>
+    <Reference Include="PresentationCore" />
```

Comment:
I don't think this is ok - if I am understanding correctly this binary is WPF and now DynamoCore depends on it. We'll have to find another way. What did you need this for?

Reply:
It seems like maybe this can just be reverted?

Reply:
yep